WorksheetB

```diff
@@ -0,0 +1,39 @@
+# Fallout word guessing game
+import random 
+correct word =  pick random word from (selection words of all 5 char long)
+position one character = corrrect answer 1
+position two character = corrrect answer 2
+position three character = corrrect answer 3
+position four character = corrrect answer 4
+position five character = corrrect answer 5
+likeness = 0
```

Comment:
This should be inside the while loop, otherwise the likeness score will accumulate over subsequent guesses